packages/platform-server/test/event_replay_spec.ts

```diff
@@ -8,15 +8,16 @@
 
 import {DOCUMENT} from '@angular/common';
 import {Component, destroyPlatform, getPlatform, Type} from '@angular/core';
-import {bootstrapEventContract, EventContract} from '@angular/core/primitives/event-dispatch';
+import {EventContract} from '@angular/core/primitives/event-dispatch';
 import {TestBed} from '@angular/core/testing';
 import {bootstrapApplication, provideClientHydration} from '@angular/platform-browser';
 import {withEventReplay} from '@angular/platform-browser/src/hydration';
+import * as fs from 'fs';
```

Comment:
Avoid using namespace imports use direct imports, also in this case you can use promises

```suggestion
import { readFile } from 'node:fs/promises';
```